NEWS.md

```diff
@@ -1,3 +1,15 @@
+## CHANGES WITH 34:
```

Comment:
Can you bring line 22 on top of this file? This was my fault when I did the last release and I forgot to fix it, sorry. :(